src/applications/static-pages/view-modify-dependents/view-dependents-cta/containers/ViewDependentsCTA.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+const ViewDependentsCTA = props => {
+  let content;
+  if (props.includedInFlipper === undefined) {
+    content = <LoadingIndicator message="Loading..." />;
+  } else if (props.includedInFlipper === false) {
+    content = (
+      <a
+        className="usa-button-primary va-button-primary"
+        href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=dependent-compensation"
+      >
+        Go to eBenefits to add or modify a dependent
+      </a>
+    );
+  } else {
+    content = (
+      <a
+        href="/disability/view-dependents/"
```

Comment:
This should be updated to `/view-change-dependents/add-remove-form-686c/`

Reply:
This button is supposed to go to the View Dependents page, not the 686 form itself. There is a second flipper component I am building in ticket [8068](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8068)